package.json

```diff
@@ -75,6 +75,7 @@
     "eslint": "^8.57.0",
     "eslint-import-resolver-alias": "^1.1.2",
     "eslint-import-resolver-typescript": "^3.8.3",
+    "eslint-plugin-cypress": "^3.5.0",
```

Comment:
This is not needed, This is part of integration tests

Reply:
@brianwcook I think you forgot about this one :) we don't need to add this lib here. If you are trying to run e2e tests locally, you can check more info in the `e2e-tests/README.md` file